tests/integration/datasets/test_demo.py

```diff
@@ -24,6 +24,7 @@ def test_get_available_demos_single_table():
         'num_tables': ['1'] * 9
     })
     expected_table['size_MB'] = expected_table['size_MB'].astype(float).round(2)
+    expected_table['num_tables'] = expected_table['num_tables'].astype(int)
```

Comment:
I tried changing the astype call to cast to float here and the test still passed, so I think to ensure `tables_info['num_tables']` is an integer dtype we should add a new check that tests directly using `pd.api.types.is_integer_dtype` to confirm it. (and same deal for the other test in this file)

Reply:
Actually this wasn't even testing the column type, just adding a column as a specific type. Added an assert to confirm that the column is of integer type using `pd.api.types.is_integer_dtype`